draft-ietf-httpbis-origin-frame.md

```diff
@@ -153,6 +153,20 @@ The Origin Set is also affected by the 421 (Misdirected Request) response status
 MUST create the ASCII serialisation of the corresponding request's origin (as per {{!RFC6454}},
 Section 6.2) and remove it from the connection's Origin Set, if present.
 
+Note:
+
+: Using an alternative service {{?RFC7838}} can result in using a service that has a different
+  port to the original service (that is, the server that a client contacts when it has no active
+  alternative services).  In that case, the initial origin will have a different port number.  A
+  client could be unable to send requests for the origin that advertised the alternative
+  service.  Explicitly including origins in the ORIGIN frame avoids this problem.
+
```

Comment:
> When sending an ORIGIN frame to a connection that is initialised as an Alternative Service {{?RFC7838}}, the initial origin set {{ref}} will contain an origin with the appropriate scheme and hostname (since Alternative Services specifies that the origin's hostname be sent in SNI). However, it is possible that the port will be different than that of the intended origin, since the initial origin set is calculated using the actual port in use, which can be different for the alternative service. In this case, the intended origin ought to be sent in the ORIGIN frame explicitly.